src/github.com/getlantern/balancer/dialer.go

```diff
@@ -131,7 +131,7 @@ func (d *dialer) defaultCheck() bool {
 			Dial: d.Dial,
 		},
 	}
-	ok, timedOut, _ := withtimeout.Do(10*time.Second, func() (interface{}, error) {
+	ok, timedOut, _ := withtimeout.Do(60*time.Second, func() (interface{}, error) {
```

Comment:
Now that we are changing this, what do you think of making it a local constant instead of a hardcoded value? Maybe `proxyHealthTimeout` or `defaultCheckTimeout` instead of the hardcoded value that depends on the context to be understood. I think it will be easier to read and to change.